sync/ui.md

```diff
@@ -20,16 +26,41 @@ There are 4 main screens in the initial Admin UI:
 
 ### Data Schema
 
-x
+*TODO*: why a resolver per property (see screenshot)? Should it not be one resolver per type?
```

Comment:
ping @david-martin

Reply:
I don't fully understand how these "resolvers" do but according to Apollo docs, [... a schema needs to have resolve functions for all fields](https://www.apollographql.com/docs/graphql-tools/resolvers#Resolver-map).

Reply:
a resolver is typically defined for all Queries and Mutations.
However, a field level resolver makes sense for a 1:M relationship.
for example, given then below schema,

```
type Note {
  id: String
  content: String
  comments: [Comment]
}

type Comment {
  id: String
  text: String
  note_id: String
}
```

a resolver would be attached to the `comments` field of `Note` for resolving all `Comments` that are linked to that Note e.g. execute a SQL query like 'select * from Comments where note_id = "some_id"'